### PR TITLE
Simplify tests

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Review by ToddGrun — APPROVED** [review]
:shipit:

File: src/Features/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ static Test()
             public Test()
             {
                 _sharedState = new SharedVerifierState(this, DefaultFileExt);
+                this.FixedState.InheritanceMode = StateInheritanceMode.AutoInherit;
```
Comment (sharwell) [inline src/Features/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs:54]: I am highly concerned about this significantly changing the test behavior in ways that cannot be explained.

Comment (CyrusNajmabadi) [reply src/Features/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs:54]: What is the significant change in test behavior.  

> that cannot be explained.

Who can explain it?  Isn't this the behavior of normal CodeFix Tests?  They inherit the end state from the start state?  Which is why you don't need to supply the 'FixedCode' if it is the same as the 'TestCode'.

**Comment by CyrusNajmabadi** [comment]
Going to mvoe forward on this, esp. as it mirrors how code fixes already work, unless tehre is a demonstrable problem here.  Being able to just supply TestCode, and nothing else, to indicate ethat nothing changes is much nicer.  It also allows the test to just be a single declarative expression, instead of needing to define locals which are passed multiple times to the .Test constructor.  This is the form we use everywhere for code-fixes, so having parity with refactorings is good.

I'll wait till tomorrow and merge then if there's no additional information added.

**Pull request merged**